### DISCUSSION TITLE
Wand Cage updates?

### DISCUSSION THREAD

**Discussion by nomakewan** [gpstar81/GPStar-proton-pack#237] [Ideas]
I have to say that the v3 Wand Cage is pretty fantastic; it fixed an issue I had with the v2 where the wand board wouldn't quite fit right. I do have two suggestions, however.

First, it appears that the width of the larger pegs that slot into the WAV Trigger are almost exactly 0.2mm oversized. Measuring with calipers I got about 3.23mm for the width of the holes in the WAV Trigger, but about 3.43mm for the width of the pegs on the v3 Wand Board. I assume this was intentional to make it a tighter fit, but 0.2mm oversized might be a bit _too_ tight.

Second, after looking at how the wand board sits in the cage, would it be possible to add a peg to the underside corner to engage with the mounting hole drilled into the wand board? I think a peg there might help stabilize the far end of the board that isn't being held by the "shelf". Picture of where I'm suggeting the peg here:

![wandcage](https://github.com/gpstar81/GPStar-proton-pack/assets/11485516/68462cea-2e9e-462b-8feb-fb54216a5d97)

What do you think?

**Comment by gpstar81** [comment]
Did you print the cage in PET-G or PLA? We've been printing them in PET-G for awhile and it's a good fit on the pegs (more flexible, softer material?). However Jonathan has a few in PLA and they are much tighter like you mentioned. A revision just for PLA prints will be uploaded soon to compensate for the peg tightness. 

We'll look into the bottom peg for the wand board at the same time.

**Reply by nomakewan** [reply]
Ah, that explains it; I actually had it printed in PA12, which is niiiice and solid. That definitely explains things.

Thanks a bunch!
